README.md

```diff
@@ -1 +1,46 @@
-# 2024-b-grass-qgis-colors-convert
\ No newline at end of file
+# 2024-b-grass-qgis-colors-convert
+![workflow](https://github.com/ctu-fgis/2024-b-grass-qgis-colors-convert/actions/workflows/test_qgis_color_func.yml/badge.svg)
+[![License: GPL v3](https://img.shields.io/badge/License-GPLv3-blue.svg)](https://www.gnu.org/licenses/gpl-3.0)
+<br>
+**GRASS GIS Color Table Import** <br>
+![plugin icon](https://github.com/jehlijos/GRASS-GIS-Q-GIS-color-table-conversion-BACKUP/blob/main/icon.png?raw=true)
+<br>
+QGIS plugin to load predefined or custom GRASS GIS raster color tables.
+> [!NOTE]  
+> In this version, the plugin correctly transforms only tables for rasters with absolute values (like Corine Land Cover). <br> For tables with percentage values, absolute values are assigned.
+
+## Table of Contents
+- [How to Install QGIS Plugin](#how-to-install-qgis-plugin)
+- [How to Use Plugin](#how-to-use-plugin)
+- [Important Files Description](#important-files-description)
+
+## How to Install QGIS Plugin
+- Clone this repository:
+    ```
+    git clone https://github.com/ctu-fgis/2024-b-grass-qgis-colors-convert
+    ```
+- Add to QGIS: <br>
```

Comment:
Add information how to create zip file. From GitHub directly (in this case why to clone reposiotry)? Or by `pb_tool` ?